### ISSUE TITLE
VBA Regional Offices Pilot MVP

### ISSUE THREAD

**Issue by davidmpickett** [department-of-veterans-affairs/va.gov-cms#10514]
## Background
Capturing the larger tasks and their rough order of operations to complete VBA MVP

Edit: 10/30/23 Sub-epic tickets for Pilot MVP moved under this epic for current state status clarity

### User Story or Problem Statement

As a Sitewide Facilities team member, I want to see the plan for executing tasks for completing MVP.

### Definition of Done for MVP

- 7 MVP ROs are entering content into Drupal and at least 1 site has entered Dual-State
- Feedback from MVP rollout is gathered and synthesized into actionable tickets for [Fast Follow phase](https://github.com/department-of-veterans-affairs/va.gov-cms/issues/14477) & National rollout

### VA.gov Front End Pipeline 
FE design and build governed by [Platform Collab Cycle](https://github.com/department-of-veterans-affairs/va.gov-team/issues/46029)

---

### Drupal CMS Pipeline 

CMS design and build governed by [CMS Collab Cycle](https://github.com/department-of-veterans-affairs/va.gov-cms/issues/10241)

---

### Service Taxonomy Pipeline

VA Service Taxonomy work governed by CAIA & [Taxonomy Council](https://github.com/department-of-veterans-affairs/va.gov-team/tree/master/products/facilities/va-service-taxonomy#contacts--content-governance)

**Comment by EWashb** [comment]
@dawnpruitt I suggest getting this epic refined before starting the work on the ticket. It seems pretty blank right now and I just want to watch out for what the intention of this work is, the scope, and what our definition of done is :)

**Comment by jilladams** [comment]
7/20 team refinement: notes from today, @mmiddaugh @davidmpickett @swirtSJW @omahane @maxx1128 @eselkin @thejordanwood @cindymerrill @ALogsdon3 

## Organization
* #13029 is a parent to this MVP effort
* Any non-MVP tickets / epics can move to #14477 
    * TODO: 
* Any MVP issues / Tickets need to live under here, preferably organized into chunks of work that can be mapped as a gantt. Epics that exist today (TODO: add #s): 
   * Remaining Spec work
   * FE design 
   * FE research
   * CMS design
   * CMS research (tbd, decisions)
   * Dev work: 
       * TODO: We need to break this up into logical chunks (CMS vs. FE? Editor- vs. Veteran-facing?)

All VBA tickets by search VBA term (not foolproof): https://github.com/department-of-veterans-affairs/va.gov-cms/issues/14436#workspaces/sitewide-facilities-639f5253e4b702a32376339e/board

## MVP: how to ship something Editor or Veteran facing faster
We are at the 1 year mark of the project. Delays are inevitable. Still: we cannot wait another year to ship something. 
**“How small is the M in MVP?”** How could we break up delivery?

**BIG PICTURE DEPENDENCIES**
  * Finalize content specs
    * CMS: Build skeleton content types
  * Design of CMS UX
    * FE: Template FE
  * Hook up FE to CMS data

**Some options for shipping:**
1. SANDY'S DB: make the data Veteran facing
  * Create FE templates for data coming from Sandy’s DB (address, name, hours, operating status).
    * Would act like a Facility Locator detail page
    * Doesn’t inherently seem valuable

2. Break up bigger delivery 
  * Design analysis at Midpoint: Evaluate: Which portions of design are A) stable / build-ready, vs. B) Will be assessed/ affected by usability testing? For stable portions, evaluate opp to start build.
  * Could parallelize 


## NEXT STEPS:
* UX Refinement
    * Can Alexis help with Specs? She's willing.
    * Review plans for Editor research. Necessary? How can we move it faster? 
  
* FE & Drupal refinement: 
    * look over existing tickets
    * Break into logical epics 
    * Figure out what can be parallelized / dependencies.

* Permissions
    * SPIKE ticket: prove we can add a 2nd workflow without breaking things, in order to persuade CMS Team
        * Blocking permissions proposal.
    * Permissions proposal: (TODO: link Kamari's doc) Revive, refresh post-SPIKE and schedule time with CMS team.
    * Workflow / permissions changes are NOT an MVP blocker. Very nice to have, our future selves will thank us.

**Comment by jilladams** [comment]
Updated flow with notes from today's refinement and UX refinement meetings. THIS IS GETTING SHAPELY, Y'ALL.

**Comment by jilladams** [comment]
@xiongjaneg stubbed 2 launch tickets we can use for notes as we go: 
* CMS MVP pilot launch: https://github.com/department-of-veterans-affairs/va.gov-cms/issues/16445
* Full MVP launch (remaining editors + FE): https://github.com/department-of-veterans-affairs/va.gov-cms/issues/16446

**Comment by davidmpickett** [comment]
## Closed as not planned
- [Social links do not match Figma](https://github.com/department-of-veterans-affairs/va.gov-cms/issues/18217)
- [“Can’t find services” alert text doesn’t match Figma](https://github.com/department-of-veterans-affairs/va.gov-cms/issues/18216)
- [Intro text doesn’t match Figma](https://github.com/department-of-veterans-affairs/va.gov-cms/issues/18211)
- [Extremely minor grammar adjustment to service options](https://github.com/department-of-veterans-affairs/va.gov-cms/issues/18215)

## Suggest blocking of Staging Review [#16268](https://github.com/department-of-veterans-affairs/va.gov-cms/issues/16268)
- [Meta description has placeholder content for all facilities](https://github.com/department-of-veterans-affairs/va.gov-cms/issues/18226)
- [Not all published services are showing on the FE](https://github.com/department-of-veterans-affairs/va.gov-cms/issues/18214)
- [Appointment default text showing different in the front end and Drupal](https://github.com/department-of-veterans-affairs/va.gov-cms/issues/17999)
- [CTA is not showing for Spotlights](https://github.com/department-of-veterans-affairs/va.gov-cms/issues/18235) 

## Suggest blocking of Launch [#16445](https://github.com/department-of-veterans-affairs/va.gov-cms/issues/16445)
- [Pre-filtered VBA locations link](https://github.com/department-of-veterans-affairs/va.gov-cms/issues/18219)
- [Order of services within categories seems inconsistent/unclear](https://github.com/department-of-veterans-affairs/va.gov-cms/issues/18213)
- [IDs on VBA facility services are not friendly](https://github.com/department-of-veterans-affairs/va.gov-cms/issues/17986)
- [Prepare for your Visit header/content should conditionally display based on accordion items](https://github.com/department-of-veterans-affairs/va.gov-cms/issues/17852)
- [Social icons should be updated to use va-icon](https://github.com/department-of-veterans-affairs/va.gov-cms/issues/17886)
- [“Select a topic” content does not display for Service categories](https://github.com/department-of-veterans-affairs/va.gov-cms/issues/17601)
- [Additional info content is displaying in italics](https://github.com/department-of-veterans-affairs/va.gov-cms/issues/17686)
- [Custom street address is not displaying on facility service or service region](https://github.com/department-of-veterans-affairs/va.gov-cms/issues/17973)
- [Contact info phone ext field does not display on FE](https://github.com/department-of-veterans-affairs/va.gov-cms/issues/18220)
- [Appt phone numbers are not displaying in the order they appear in Drupal](https://github.com/department-of-veterans-affairs/va.gov-cms/issues/18228)
- [Overall heading structure needs to be adjusted/fixed](https://github.com/department-of-veterans-affairs/va.gov-cms/issues/18222)
  - Might need to be broken into a few different tickets

## Suggest for Fast Follow / National [#14477](https://github.com/department-of-veterans-affairs/va.gov-cms/issues/14477)
- Service Regions do not block local editors from entering content or launching their local pages. Need to have discussion around how they will be added and by whom. 
  - [On Service region, if Facility hours is selected, nothing shows on the FE](https://github.com/department-of-veterans-affairs/va.gov-cms/issues/18221)
  - [Service regions do not display service option modalities properly](https://github.com/department-of-veterans-affairs/va.gov-cms/issues/18227)

## Systemic cross-product issues
- [Expandable alert - error state has been removed from guidance](https://github.com/department-of-veterans-affairs/va.gov-cms/issues/18225)
- [Overall heading structure needs to be adjusted/fixed](https://github.com/department-of-veterans-affairs/va.gov-cms/issues/18222)
- Will all “New tab” issues be resolved by [Public Websites ticket](https://github.com/department-of-veterans-affairs/va.gov-team/issues/81506)?
  - [Main Directions link/image should not open in a new tab](https://github.com/department-of-veterans-affairs/va.gov-cms/issues/18212)
  - [Nearby locations directions links should not open in a new tab](https://github.com/department-of-veterans-affairs/va.gov-cms/issues/18218)
  - [VERA links should not open in a new tab](https://github.com/department-of-veterans-affairs/va.gov-cms/issues/18004)

**Comment by jilladams** [comment]
Proposed cut lines, not yet final ^^, tagged for @Agile6MSkinner to take next pass review (in [Slack](https://dsva.slack.com/archives/C0FQSS30V/p1717190957464869?thread_ts=1716583534.742069&cid=C0FQSS30V)).

**Comment by jilladams** [comment]
Re: new tab issues: we'll need to test and verify after PW ships the https://github.com/department-of-veterans-affairs/va.gov-team/issues/81506 change, theoretically in this sprint.

**Comment by jilladams** [comment]
From onsite: re: blockers to FE launch:
* Staging Review
  * Final staging review blockers are in Sprint 6
  * Dave is running Foundational Testing. 
* We need design review
* We need a final surgical round of QA to verify data pull through from Drupal to FE. This can happen as part of Design review. 

cc @mmiddaugh @davidmpickett @laflannery @Agile6MSkinner

**Comment by Agile6MSkinner** [comment]
Moving the collab cycle epic National Rollout and closing
